modbus/modbustcpmaster.h

```diff
@@ -37,6 +37,8 @@
 #include <QTimer>
 #include <QUuid>
 
+Q_DECLARE_LOGGING_CATEGORY(dcModbus)
```

Comment:
Maybe name it ModbusTcp, since the Modbus category is part of the core now, so we can distinguish between tcp and rtu.

Reply:
changed to ModbusTcp